code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -1,3 +1,5 @@
+#define CULT_STRUCTURE_COOLDOWN 60 SECONDS
+
```

Comment:
I really dont think this define is necessary but if you are gonna add it, it needs to be specific that it pertains to when the structure is built

again I think the code is more readable without this

Reply:
so revert it back to just 60 seconds?

Reply:
cooldowntime = world.time + 60 SECONDS
would probably be fine or
cooldowntime = world.time + 1 MINUTES

Reply:
either change it back to `1 MINUTES` as suggested or add an autodoc explanation (autodoc is the thing you see when you hover your mouse over autodoc'd variables) to it so people can know what it is at a glance, with:

```suggestion
/// The amount of time necessary for a structure to be able to produce items after being built
#define CULT_STRUCTURE_COOLDOWN 60 SECONDS

```

Reply:
Agreed, please implement mira's comment on this